documentation/modules/exploit/windows/misc/remote_mouse_rce.md

```diff
@@ -0,0 +1,114 @@
+## Vulnerable Application
+
+This module utilizes the Remote Mouse Server by Emote Interactive protocol
+to deploy a payload and run it from the server.  This module will deploy
+a payload regardless if server authentication is required.
+Tested against 4.110, current at the time of module writing
+
+Version 4.110 can be downloaded from https://www.remotemouse.net/downloads/RemoteMouse.exe
+    
+## Verification Steps
+
+1. Install the application
+2. Start msfconsole
+3. Do: `use exploit/windows/misc/remote_mouse_rce`
+4. Set `rhost` and `lhost` as required.
+5. Do: `run`
+6. You should get a shell as the user who is running Remote Mouse.
+
+## Options
```

Comment:
Could some details about `PATH` and `CLIENTNAME` (if used) be added here as well?

Reply:
added PATH